### DISCUSSION TITLE
Unreleased content

### DISCUSSION THREAD

**Discussion by flavredguccidip** [DarkflameUniverse/DarkflameServer#1637] [Q&A] [answered]
Is it possible to play unreleased content like the fire temple or possible ways to get it implemented into DLU?

**Comment by aronwk-aaron** [comment, marked as answer]
yes, implementing the script for fire temple is on the list of things to do, unfortunately it requires re-writing some major underlying parts of the codebase to make it work due to that make using more complex mechanisms than the rest of the game

**Reply by flavredguccidip** [reply]
Thank you, any idea on an ETA for this project? I remember being so excited as a kid to play it and the other dungeons upon reading about it so this would be an awesome addition.

**Reply by aronwk-aaron** [reply]
No eta
